models/lasertagger/tagging_converter.py

```diff
@@ -28,36 +27,40 @@
 import tagging
 import utils
 
-from typing import Iterable, Mapping, Sequence, Set, Text, Tuple
 
 
 class TaggingConverter(object):
-  """Converter from training target texts into tagging format."""
-
-  def __init__(self, phrase_vocabulary, do_swap=True):
-    """Initializes an instance of TaggingConverter.
+    """Converter from training target texts into tagging format."""
+    def __init__(self,
+                 phrase_vocabulary,
+                 do_swap=True,
+                 arbitrary_reordering=True):
+        """Initializes an instance of TaggingConverter.
 
     Args:
```

Comment:
please add the description for your new args

Reply:
I've added the descriptions and some more comments to explain things better